docs/content/guides/columns/column-filter.md

```diff
@@ -155,765 +449,3045 @@ import 'handsontable/dist/handsontable.full.min.css';
+    <option value="0">Brand</option>
+    <option value="1">Model</option>
+    <option value="2">Price</option>
+    <option value="3">Date</option>
+    <option value="4">Time</option>
+    <option value="5">In stock</option>
+  </select>
+</div>
+<div class="controlsQuickFilter">
+  <input id="filterField" type="text" placeholder="Filter" />
+</div>
+<br />
+<div id="exampleQuickFilter"></div>
+```
+
+```js
+import Handsontable from 'handsontable';
+import 'handsontable/dist/handsontable.full.min.css';
+
+const container = document.querySelector('#exampleQuickFilter');
+const filterField = document.querySelector('#filterField');
+
+const handsontableInstance = new Handsontable(container, {
+  data: [
+    {
+      brand: 'Jetpulse',
+      model: 'Racing Socks',
+      price: 30,
+      sellDate: '11/10/2023',
+      sellTime: '01:23',
+      inStock: false,
+    },
+    {
+      brand: 'Gigabox',
+      model: 'HL Mountain Frame',
+      price: 1890.9,
+      sellDate: '03/05/2023',
+      sellTime: '11:27',
+      inStock: false,
+    },
+    {
+      brand: 'Camido',
+      model: 'Cycling Cap',
+      price: 130.1,
+      sellDate: '27/03/2023',
+      sellTime: '03:17',
+      inStock: true,
+    },
+    {
+      brand: 'Chatterpoint',
+      model: 'Road Tire Tube',
+      price: 59,
+      sellDate: '28/08/2023',
+      sellTime: '08:01',
+      inStock: true,
+    },
+    {
+      brand: 'Eidel',
+      model: 'HL Road Tire',
+      price: 279.99,
+      sellDate: '02/10/2023',
+      sellTime: '13:23',
+      inStock: true,
+    },
+  ],
+  columns: [
+    {
+      title: 'Brand',
+      type: 'text',
+      data: 'brand',
+    },
+    {
+      title: 'Model',
+      type: 'text',
+      data: 'model',
+    },
+    {
+      title: 'Price',
+      type: 'numeric',
+      data: 'price',
+      numericFormat: {
+        pattern: '$ 0,0.00',
+        culture: 'en-US',
+      },
+    },
+    {
+      title: 'Date',
+      type: 'date',
+      data: 'sellDate',
+      className: 'htRight',
+    },
+    {
+      title: 'Time',
+      type: 'time',
+      data: 'sellTime',
+      correctFormat: true,
+      className: 'htRight',
+    },
+    {
+      title: 'In stock',
+      type: 'checkbox',
+      data: 'inStock',
+      className: 'htCenter',
+    },
+  ],
+  colHeaders: true,
+  height: 'auto',
+  filters: true,
+  className: 'exampleQuickFilter',
+  licenseKey: 'non-commercial-and-evaluation',
+});
+
+// add a filter input listener
+filterField.addEventListener('keyup', function (event) {
+  const filters = handsontableInstance.getPlugin('filters');
+  const columnSelector = document.getElementById('columns');
+  const columnValue = columnSelector.value;
+
+  filters.removeConditions(columnValue);
+  filters.addCondition(columnValue, 'contains', [event.target.value]);
+  filters.filter();
+
+  handsontableInstance.render();
+});
+```
+
+```css
+.controlsQuickFilter {
+  margin: 0;
+  padding: 0;
+}
+
+.selectColumn {
+  color: white;
```

Comment:
This line makes this element invisible when its opened in jsfiddle by "Edit" button. 
<img width="524" alt="Screenshot 2023-04-25 at 14 09 25" src="https://user-images.githubusercontent.com/30186818/234273362-33be74dd-c00f-4e56-9eef-9d5b35aeefc5.png">

Reply:
When considering the accessibility of documentation, I would suggest increasing the low contrast of gray text by changing it based on the theme color. 

```
.selectColumn {
  color: black;
  @media screen and (prefers-color-scheme: dark) {
  color: white;
}

```
This has worked well for me on Jfiddle, where the color was adjusted based on my system settings.

Reply:
Nevermind my proposition, it doesn't work in documentation. We can stay with `gray`.